content/actions/reference/workflow-syntax-for-github-actions.md

```diff
@@ -542,11 +542,25 @@ jobs:
         uses: docker://alpine:3.8
 ```
 
-#### Example using a Docker public registry action
+##### Example using the {% data variables.product.prodname_github_container_registry %}
 
 `docker://{host}/{image}:{tag}`
 
-A Docker image in a public registry.
+A Docker image in the {% data variables.product.prodname_github_container_registry %}.
+
+```yaml
+jobs:
+  my_first_job:
+    steps:
+      - name: My first step
+        uses: docker://ghcr.io/cloud-builders/gradle
```

Comment:
We could use the existing example from the Packages docs: https://docs.github.com/en/free-pro-team@latest/packages/guides/pushing-and-pulling-docker-images#pulling-container-images
```suggestion
        uses: docker://ghcr.io/OWNER/IMAGE_NAME
```